widget_driver/example/lib/widgets/coffee_community/coffee_library/coffee_detail/coffee_detail_page_driver.dart

```diff
@@ -9,8 +8,10 @@ part 'coffee_detail_page_driver.g.dart';
 class CoffeeDetailPageDriver extends WidgetDriver {
   final Coffee _coffee;
 
-  CoffeeDetailPageDriver(BuildContext context)
-      : _coffee = context.read<Coffee>(),
+  CoffeeDetailPageDriver(
+    BuildContext context, {
+    @driverProvidableProperty required Coffee coffee,
```

Comment:
Maybe also add some example where we use the positional argument.. and the this.variable name.. to show users how this works